packages/block-library/src/pullquote/blockquote.native.scss

```diff
@@ -1,8 +1,8 @@
 .quote {
-	font-size: 18px;
+	font-size: 1.125em;
 }
 
 .citation {
-	font-size: 14px;
-	margin-top: 12px;
+	font-size: 0.875em;
+	margin-top: 0.85em;
 }
```

Comment:
I'm not sure if the values under `.citation` is correct, because I'm not sure if `.citation` is nested under `.quote`. It's not really clear to me what's going on in the `native.js` file for this block.

Reply:
This part wasn't 100% clear to me either... So I opted for the safest value, it made sense :sweat_smile:

Reply:
Native mobile doesn't support `em` at all so it causes a crash on pull-quote. We reverted `*.native.scss` changes here: https://github.com/WordPress/gutenberg/pull/24936

Reply:
Ah I didn't know that, thank you!